### DISCUSSION TITLE
Possibility to disable signing up?

### DISCUSSION THREAD

**Discussion by sylvarg** [Erudika/scoold#457] [Q&A]
Hi there. I'm (slowly) discovering scoold. 
I would like to avoid any unwanted user to sign up during my testing phase, while still allowing legitimate users to sign in. I'm just using basic login/mail/passwd login. Is this something possible? Thanks.

**Comment by albogdano** [comment]
You have a couple of options:
- add `scoold.approved_domains_for_signups = "domain.com"` to your configuration
- disable password auth entirely `scoold.password_auth_enabled = false`

Fore details check the docs on all configuration options https://github.com/Erudika/scoold?tab=readme-ov-file#configuration

**Reply by sylvarg** [reply]
I have already tested these options, but since I can not set up email verification on my installation (at least, for now) :
- option 1 will allow anyone knowing the correct domain (which will be very easy to guess) to create an account very easily ;
- option 2 completely disables the ability for regular, already registered, users to log in.

Correct me if I'm wrong, but I didn't spot any configuration options simply disabling new user sign up?

**Reply by albogdano** [reply]
There could be a new option `scoold.signups_enabled` but it is yet to be implemented. For now, you could use some cryptic made-up domain name. If you are the admin, you should be able to log in with an email, which is not from that domain.

**Reply by sylvarg** [reply]
OK. Using an ad-hoc domain name is what I had in mind, but having the possibility to entirely disable users signing up would be a very great feature which would be very helpful, at least for security reasons. Hope to see it implemented soon.
Thanks!
